words/tails.txt

```diff
@@ -272,3 +272,12 @@ qilin
 fluffy
 bun
 mercat
+bison
+buffalo
+gaur
+saola
+anoa
+banteng
+beefalo
+yak
+cow
```

Comment:
These last two are duplicates, we have a test that checks for duplicates in the lists. Can you run the test and remove any duplicates?

Reply:
Wow, apologies -- GitHub search failed me here.